src/libs/isReportMessageAttachment.ts

```diff
@@ -13,10 +15,21 @@ export default function isReportMessageAttachment(message: Message | undefined):
         return false;
     }
 
-    if (message.translationKey && message.text === CONST.ATTACHMENT_MESSAGE_TEXT) {
-        return message?.translationKey === CONST.TRANSLATION_KEYS.ATTACHMENT;
+    if (message.translationKey) {
+        return message.text === CONST.ATTACHMENT_MESSAGE_TEXT && message.translationKey === CONST.TRANSLATION_KEYS.ATTACHMENT;
+    }
+
+    const hasAttachmentHtml = attachmentRegex.test(message.html) || message.html === CONST.ATTACHMENT_UPLOADING_MESSAGE_HTML;
```

Comment:
these are being executed left to right - I think you can also swap them around to avoid running a regexp check when a simple strict equality check on `message.html === CONST.ATTACHMENT_UPLOADING_MESSAGE_HTML;` will be enough.

Reply:
that way we can mostly rely on equality checks, and only run a `regexp.test()` or `isVideo()` as fallbacks.

Reply:
good job on this one!

Reply:
approving it already, this is a minor change